policy/modules/system/systemd.if

```diff
@@ -2290,16 +2292,16 @@ interface(`systemd_map_resolved_exec_files',`
 
 ########################################
 ## <summary>
-##	Send and receive messages from
-##	systemd resolved over dbus.
+##	Exchange messages with
+##	systemd resolved over dbus or varlink.
```

Comment:
This is just a question: Are current rules sufficient to use varlink?

Reply:
If "current" means "this patch", then yeah, it seems to work OK.